### PR TITLE
Feature/milestone 1

### PR DESCRIPTION
**Pull request by Israel-Laguan**
# Resume

We have created the methods as described in the project requirements, added some basic tests, and *came to the conclusion that we met all of the requirements*.

# Proposal

we want to have a review of the code to know if we need to make any changes. If no changes needed we want the project to be ACCEPTED so we can move to the next project with a happy smile :)

# Comments

Working on ruby is interesting because of new concepts, but is a good thing that ruby have a lot of good utilities that helps a lot. It is an interesting language to work with.

Besides that working in organizing files and folders was fun.

### CODE REVIEW

File: lib/tic_tac_toe.rb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# frozen_string_literal: true
+
+p 'Hello world!'
```
Comment (bolah2009) [inline lib/tic_tac_toe.rb]: Let's delete this

> `main.rb` file should be responsible for user interaction. This is the only one place where `puts` and `get.chomp` can be used.

**Review by bolah2009 — APPROVED** [review]
### Hi Team :wave: 

Nice job making the `main.rb` file executable. :clap: 

Please prevent the use of `puts`, `p`, `gets`, `prints` or any other user interaction from the `lib` directory in future milestones. 

#### It's a :+1: from me. 

Let's move to Milestone 2

#### :heavy_check_mark: Approved.

File: lib/tic_tac_toe.rb (hunk 1 above)
Comment (Israel-Laguan) [reply lib/tic_tac_toe.rb]: Sure man, thanks for the review!

Comment (maelfosso) [reply lib/tic_tac_toe.rb]: Please, it's still there

**Review by maelfosso — CHANGES_REQUESTED** [review]
Hello @Israel-Laguan @izaiasneto4 ,

Please, you didn't make the changes suggested by @bolah2009 

Kindly do and submit a new code review request.
Thanks.

**Mael FOSSO**
[Github](https://github.com) | [Twitter](https://twitter.com) | [LinkedIn](https://www.linkedin.com/in/mael-elvis-fosso-650b6346/)

**Review by whiz25 — APPROVED** [review]
## Good job implementing the changes
It's a :+1: from me.

### Martin Afani

**Pull request merged**